public/demo.js

```diff
@@ -367,9 +374,9 @@ $(document).ready(function () {
     // Fallback: search for a language-only match (e.g. en to es)
     source = source.split('-').shift(); // split around '-' to handle cases like arz where the language part is actually 3 letters
     target = target.split('-').shift();
-    for (var y in modelList) {
+    for (y in modelList) {
```

Comment:
This was actually a fix for a bug I introduced the other day - I double-defined `y` (and it failed the linter - see #41). The linter would have failed this PR if `y` wasn't declared at all. 

However, the better fix was to rename `y` to a new var name, not re-use the `y` from above. (I would have merged the fix right away, but I was asked to wait.) 

Now that #41 is merged, this change is no longer needed